osu.Game/Overlays/Pause/PauseOverlay.cs

```diff
@@ -69,6 +69,9 @@ public int Retries
         protected override void PopIn() => FadeIn(transition_duration, EasingTypes.In);
         protected override void PopOut() => FadeOut(transition_duration, EasingTypes.In);
 
+        // Don't let mouse down events through the overlay or people can click circles while paused.
+        protected override bool OnMouseDown(InputState state, MouseDownEventArgs args) => true;

```

Comment:
pretty sure this was left out because of the issues we have with multiple input managers.